app/models/creatorship.rb

```diff
@@ -45,8 +45,9 @@ def check_invalid
   # disabled invitations.
   def check_disallowed
     return if approved? || pseud.nil?
+    # Prevents this error from appearing in addition to the errorfor banned or suspended users. 
```

Comment:
Little typo: `errorfor`

Reply:
hah. That one got by me. I'll fix that also when i work on the other code!